.gitmodules

```diff
@@ -6,4 +6,5 @@
 	url = https://github.com/Xilinx/cmakeModules.git
 [submodule "runtime_lib/xaiengine/aie-rt"]
 	path = runtime_lib/xaiengine/aie-rt
-	url = https://github.com/stephenneuendorffer/aie-rt.git
+	url = https://github.com/eddierichter-amd/aie-rt.git
```

Comment:
Is this repo/branch up to date with https://github.com/stephenneuendorffer/aie-rt/tree/phoenix_v2023.2?

Reply:
Yes it is up to date and I have a PR which still needs to go through. Ideally, we can just point to Steve's: https://github.com/stephenneuendorffer/aie-rt/pull/4

Reply:
Commented there.